Woof/Woof/Model/LocalStorage/LocalStorage.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+
+/// A persistent storage designed for managing a collection of values.
+/// These values are stored and retrieved using a key that uniquely identifies the value.
+/// The key is used to find the data within the storage.
+class LocalStorage {
```

Comment:
nit:

Should we give it more specific name:

```suggestion
class KeyValueStorage {
```

or smth like that
(according to our interface)?

Reply:
I do not change it, suddenly there will be a case when we change the implementation and there will be no key-value storage

Reply:
Renamed as suggested